src/System.Runtime.Serialization.Xml/tests/DataContractSerializer.cs

```diff
@@ -1680,6 +1680,7 @@ public static void DCS_TypeWithCommonTypeProperties()
         Assert.StrictEqual<TypeWithCommonTypeProperties>(value, deserializedValue);
     }
 
+#if NET_NATIVE
```

Comment:
That might not work. The test caused failure in complie time as UWP toolchain failed to generate serializer for type `TypeWithTypeProperty`. The toolchain probably would still see the type even if the test is marked with `[SkipOnTargetFramework]`.